app/buildpacks/ruby_buildpack.rb

```diff
@@ -1,7 +1,10 @@
 class RubyBuildpack
   include Buildpack
 
-  TEMPLATE = 'FROM <%= base_images_registry_url %>/ruby:<%= ruby_version %>'.freeze
+  TEMPLATE = [
+    'FROM <%= base_images_registry_url %>/ruby:<%= ruby_version %>'
+  ].join("\n").freeze
```

Comment:
The advantage of leaving it like this is that the variable is already set for future additions to the Dockerfile keeping it in the same way as the NodeJS buildpack.

I left it like this on purpose to avoid the need to dig on the code later or to someone implementing it in any way different than on the NodeJS buildpack.